tests/MSBuildDeviceIntegration/Tests/MonoAndroidExportTest.cs

```diff
@@ -118,7 +118,6 @@ protected override void OnCreate (Bundle bundle)
 	</application >
 </manifest>";
 				Assert.True (b.Install (proj), "Project should have installed.");
-				ClearAdbLogcat ();
 				b.BuildLogFile = "run.log";
 				Assert.True (b.RunTarget (proj, "StartAndroidActivity", doNotCleanupOnUpdate: true), "Project should have run.");
```

Comment:
Should this use `RunProjectAndAssert` ?

Reply:
Yeah I think it is probably best for us to exercise our run target everywhere we can, I will try look for more instances like this.

Reply:
Fixed.